src/layout/constraint.rs

```diff
@@ -70,6 +70,23 @@ pub enum Constraint {
     /// assert_eq!(4, Constraint::Length(4).apply(10));
     /// ```
     Length(u16),
+    /// Apply no more or no less than the given amount
+    /// ```
+    /// # use ratatui::prelude::*;
+    /// assert_eq!(0, Constraint::Fixed(4).apply(0));
+    /// assert_eq!(4, Constraint::Fixed(4).apply(4));
+    /// assert_eq!(4, Constraint::Fixed(4).apply(10));
```

Comment:
I'm not exactly sure what `.apply` is supposed to do, so this docstring can be improved.

Reply:
Improved the docstring in the latest commit.